_posts/2020-05-15-issue-2.md

```diff
@@ -0,0 +1,71 @@
+---
+layout: post
+title: 'Issue 2: Who calls the shots?'
+category: Volume 2
+image:
+    file: vol-2-issue-2-header.png
+    alt: 'Screenshot of a tweet sent by Elon Musk announcing the re-opening of a Tesla factory, contra local safety regulations'
+    caption: "Elon Musk acting as if he's some sort of civil rights hero for defying regulations meant to keep workers safe"
+    source_link: 'https://twitter.com/elonmusk/status/1259945593805221891'
+    half_width: false
+preview: "Elon Musk gets to call the shots. He shouldn't."
+---
+
+Who calls the shots at Tesla Motors? Is it the workers who collectively build all the products that generate the company's wealth? No, it’s the company’s multi-billionaire CEO, who recently threw a fit on Twitter over health and safety regulations that would delay resuming production. How does he manage to get away with that?
+
+<!--excerpt-->
+
+{% include post_image.html %}
+
+Tesla CEO Elon Musk has done a great job of framing his decision to resume production as if it were a noble act on behalf of his workers. But [leaked emails from management](https://www.theguardian.com/technology/2020/may/12/tesla-workers-unemployment-benefits-emails) show that Tesla workers are being intimidated to return to work with the claim that they may be ineligible for unemployment benefits if they do not return to work. This isn't Musk taking a heroic stand to save his workers; it's a means of exercising the power that he holds over his workers during a global pandemic, so that they can keep making him money even if it puts their lives at risk.
+
+We’ve been here before. Jose Moran, a Tesla quality inspector at the plant in Fremont California, saw coworkers put in 60-70 hours each week and get injured on the job. In 2016, he reached out to the United Auto Workers for help, and in 2017 he wrote [an open letter](https://medium.com/@moran2017j/time-for-tesla-to-listen-ab5c6259fc88): “Most Tesla production workers earn between $17 and $21 hourly. The average auto worker in the nation earns $25.58 an hour, and lives in a much less expensive region.” Unfortunately, within two years, the unionizing campaign had failed. A ruling by the National Labor Relations Board (NLRB) indicated that [Tesla had illegally sabotaged organizing efforts](https://www.theverge.com/2018/9/29/17914572/tesla-union-trial-silicon-valley-unionization-elon-musk) with allegations that a union would render workers voiceless.
```

Comment:
Missing comma between "Fremont" and "California".

Reply:
fixed